docs/csharp/misc/CS4009.md

```diff
@@ -39,6 +39,9 @@ public class Example
 
 ## To correct this error
 
+> [!NOTE]
```

Comment:
I think that I would move the note up to the previous section, to follow the "You cannot use the `async` keyword..." sentence. I'd also change it from NOTE to IMPORTANT.